lib/Exporter.js

```diff
@@ -1,25 +1,22 @@
-// Exports data as any supported format 
+// Exports data as any supported format
 // take in a format, file key, geojson, and callback
-var fs = require('fs'),
-  mkdirp = require('mkdirp'),
-  async = require('async'),
-  crypto = require('crypto'),
-  projCodes = require('esri-proj-codes'),
-  mv = require('mv'),
-  spatialReference = require('./SpatialReference'),
-  rm = require('rimraf');
-
-var exec = require('child_process').exec;
-
+var fs = require('fs')
+var mkdirp = require('mkdirp')
+var async = require('async')
+var crypto = require('crypto')
+var projCodes = require('esri-proj-codes')
+var mv = require('mv')
+var spatialReference = require('./SpatialReference')
+var rm = require('rimraf')
+var exec = require('child_process').exec
 var ogrFormats = {
   kml: 'KML',
-  zip: '\"ESRI Shapefile\"',
+  zip: '"ESRI Shapefile"',
```

Comment:
I think this got automatically stripped out by a linter -- are the backslashes needed?